### PR TITLE
Makefile: alias `build` target as `bin`

### LINKED ISSUE #2529: synchronize common make targets across all projects
There should be a list of common make targets that should be the same for all projects.
For example:
- `make bin` makes binaries using host env
- `make docker/bin` -- makes binaries using clean docker env.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #2529.

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
We have a lot of `make build` dependencies: inside Dockerfile, GA files, neo-bench, neo-go-sc-wrkshop. Also. the users of neo-go could get used to `make build`. Do we need to completely remove `make build`?

**Comment by roman-khimov** [comment]
> Do we need to completely remove make build?

No, why would we do that?

File: Makefile [hunk 1]
```diff
@@ -32,10 +32,12 @@ build: deps
 		&& export CGO_ENABLED=0 \
 		&& go build -trimpath -v -ldflags $(BUILD_FLAGS) -o ${BINARY} ./cli/main.go
 
+bin: build
+
 neo-go.service: neo-go.service.template
 	@sed -r -e 's_BINDIR_$(BINDIR)_' -e 's_UNITWORKDIR_$(UNITWORKDIR)_' -e 's_SYSCONFIGDIR_$(SYSCONFIGDIR)_' $< >$@
 
-install: build neo-go.service
+install: bin neo-go.service
```
Comment (roman-khimov) [inline Makefile:40]: Unnecessary.

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ NeoGo, `:latest` points to the latest release) or build yourself.
 To build NeoGo you need Go 1.16+ and `make`:
 
 ```
-make build
+make bin
```
Comment (roman-khimov) [inline README.md:54]: Ditto.

File: README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ To build NeoGo on Windows platform we recommend you to install `make` from [MinG
 package](https://osdn.net/projects/mingw/). Then, you can build NeoGo with:
 
 ```
-make build
+make bin
```
Comment (roman-khimov) [inline README.md:65]: Ditto.

**Comment by AnnaShaleva** [comment]
> No, why would we do that?

Just clarifying the task, OK.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**